azure-kusto-ingest/azure/kusto/ingest/__init__.py

```diff
@@ -15,5 +15,8 @@
     JsonColumnMapping,
     IngestionProperties,
     IngestionMappingType,
+    ColumnMapping,
+    TransformationMethod,
```

Comment:
Not sure those Enums should be exposed

Reply:
the only reason i added them is for them to be exposed